tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -410,9 +394,13 @@ def test_it_sets_the_right_javascript_config_settings(
 
         # Test that we pass through parameters from the request made to us
         # onto the configure module item call
+        BearerTokenSchema.assert_called_once_with(pyramid_request)
+        bearer_token_schema.authorization_param.assert_called_once_with(
+            pyramid_request.lti_user
+        )
         assert form_fields == Any.dict.containing(
             {
-                "authorization": Any.string(),
+                "authorization": bearer_token_schema.authorization_param.return_value,
```

Comment:
We simply test that `BasicLTILaunchViews` sets `"authorization_param"` to what `BearerTokenSchema` returned. Testing the  implementation details of what `BearerTokenSchema` returns (a JWT encoded with a secret) is done in `BearerTokenSchema`'s own unit tests. This is much simpler than these tests decoding the JWT to a dict, doesn't duplicate existing `BearerTokenSchema` unit tests, and doesn't couple `BasicLTILaunchViews` tests to `BearerTokenSchema`'s implementation details (only its API), doesn't allow `BasicLTILaunchViews` tests to fail if there's a bug in `BearerTokenSchema`.